.github/workflows/test-local.yml

```diff
@@ -172,9 +185,10 @@ jobs:
         if: inputs.testing-minimal == false
         shell: bash
         run: |
-          python -m pip install build
-          python -m build
-          python -m pip install dist/*.whl
+          # python -m pip install build
+          # python -m build
+          # python -m pip install dist/*.whl
```

Comment:
```suggestion
```

Reply:
I am going to test building the wheels... just in case....